app/lib/limit_graph.py

```diff
@@ -0,0 +1,42 @@
+from app.lib.map_graph import map_graph
+
+def limit_graph(graph, threshold):
```

Comment:
Please add a brief comment describing how this function works

Reply:
I figured since map_graph maps the indices of connected nodes, it can be applied to other use cases when we need to further process the graph.